### PR TITLE
Add supports MSBuild.dll to GUI

### LINKED ISSUE #474: Add supports `MSBuild.dll` to GUI
Hello!

Add supports `MSBuild.dll` to GUI

I found an old closed issue and comment https://github.com/KirillOsenkov/MSBuildStructuredLog/issues/57#issuecomment-293410920, but it is now only possible to choose `MSBuild.exe` from VS. MSBuild from SDK is on the path `dotnet\sdk\<version>\`. By default, we can just use `msbuild`:
```sh
dotnet msbuild Some.sln /t:Rebuild /v:diag /nologo /clp:NoSummary;Verbosity=minimal /bl
```

### PR DESCRIPTION
**Pull request by rstm-sf**
Closes #474

### CODE REVIEW

File: src/StructuredLogViewer.Avalonia/MainWindow.xaml.cs [hunk 1]
```diff
@@ -571,6 +571,13 @@ private async Task BrowseForMSBuildExe()
                 return;
             }
 
+            var isMsBuild = fileName.EndsWith("MSBuild.dll", StringComparison.OrdinalIgnoreCase)
+                         || fileName.EndsWith("MSBuild.exe", StringComparison.OrdinalIgnoreCase);
+            if (!isMsBuild)
+            {
+                return;
```
[the remark this answers is not in the document]
Comment (KirillOsenkov) [reply src/StructuredLogViewer.Avalonia/MainWindow.xaml.cs:578]: I wouldn’t bother

File: src/StructuredLogViewer.Avalonia/BuildParametersScreen.cs [hunk 2]
```diff
@@ -64,11 +69,10 @@ private void Copy()
             //Clipboard.SetText(commandLine);
         }
 
-        private ICommand browseForMSBuildCommand;
-        public ICommand BrowseForMSBuildCommand => browseForMSBuildCommand ?? (browseForMSBuildCommand = new Command(BrowseForMSBuild));
```
Comment (rstm-sf) [inline src/StructuredLogViewer.Avalonia/BuildParametersScreen.cs:68]: We can leave this behind by implementing `AsyncCommand`

**Comment by rstm-sf** [comment]
We can use the [OmniSharp method to find](https://github.com/OmniSharp/omnisharp-roslyn/blob/09d4884d4ff1306b4da2a881c3dc62c648629916/src/OmniSharp.Host/Services/DotNetCliService.cs#L103) where the sdk is located

**Comment by rstm-sf** [comment]
Does not change msbuild in case of avalonia

**Comment by KirillOsenkov** [comment]
Mention me when this is ready for review. Thanks!

**Comment by rstm-sf** [comment]
I think it's enough now :)

**Comment by KirillOsenkov** [comment]
Thanks!

**Pull request merged**